lib/index.js

```diff
@@ -1,9 +1,17 @@
-var __VERSION__ = null;
+import Consumer from "./consumer.js"
+import LongPollTransport from "./long_poll_transport.js"
+import MultiplexedConsumer from "./multiplexed_consumer.js"
+import MultiplexedLongPoll from "./multiplexed_long_poll.js"
+import Transport from "./transport.js"
+import WebSocketTransport from "./web_socket_transport.js"
+import MultiplexedWebSocket from "./multiplexed_web_socket.js"
 
-if ("webpack" != process.env.NODE_ENV) {
-  require("coffee-script/register");
-  __VERSION__ = require("../helpers/node").version
+export {
+  Consumer,
+  LongPollTransport,
+  MultiplexedConsumer,
+  MultiplexedLongPoll,
+  Transport,
```

Comment:
Do we need to expose the `Transport` base class to consumers?

Reply:
We probably don’t need, it’s exposed just keep it available since it was available before. But you’re right, we don’t need it.